packages/contracts-adapter/script/utils/fetchBlockTimes.ts

```
@@ -0,0 +1,294 @@
+    results[chainName] = { oldBlockTime, newBlockTime }
+
+    if (newBlockTime !== null) {
+      chainsConfig[chainName].blockTime = newBlockTime
+    }
+  }
+
+  // Print summary
+  console.log(`\n${colors.bold}${colors.cyan}Summary${colors.reset}\n`)
+
+  const updated = Object.entries(results).filter(
+    ([_, r]) => r.newBlockTime !== null && r.newBlockTime !== r.oldBlockTime
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)